submitit/core/core.py

```diff
@@ -391,6 +446,12 @@ def wait(self) -> None:
         while not self.done():
             _time.sleep(1)
 
+    async def _async_wait(self, poll_interval: tp.Union[int, float] = 1) -> None:
```

Comment:
made this private so lint wouldn't complain, but it could also be public.

Reply:
I think this should be public. What is the linting error ?

Reply:
there is a lint that we can't have more than 20 public methods.

Reply:
> there is a lint that we can't have more than 20 public methods.

There is no lint that we can't deactivate :D